base/fastmath.jl

```diff
@@ -273,6 +273,9 @@ end
 
 
 # Math functions
+exp2_fast(x::Union{Float32,Float64})  = Base.Math.exp2_fast(x)
```

Comment:
Probably doesn't matter, but might it not be better to rename the fast methods in `Math` with an underscore prefix, just in case there is ever a freak naming clash? Not a direct suggestion to rename them, but just a minor concern.

Reply:
Probably a good idea, but not for this PR.